tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1691,6 +1692,26 @@ public function theDataOfTheResponseShouldMatch(
 		);
 	}
 
+	/**
+	 * @Then the JSON data of the response should match
+	 *
+	 * @param PyStringNode|string $schemaString
+	 * @param object|null $response
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function theDataOfTheResponseShouldMatch($schemaString, ?object $response = null): void {
```

Comment:
need `$response`? step implementation should only accept the args passed down from the step
```suggestion
	public function theDataOfTheResponseShouldMatch($schemaString): void {
```

Reply:
This is done because the json schema doesnot support `contains`. So the actual required response is filtered and passed here for some json match.